spec/features/companies/user_sees_all_city_locations_of_company_with_count_spec.rb

```diff
@@ -0,0 +1,21 @@
+require 'rails_helper'
+
+RSpec.describe "User visits a specific company page" do
+  scenario "and sees all cities where offices are located for company with quantity" do
+    company = Company.create!(name: "Turing")
+    location = company.locations.create!(city: "Denver")
+    location = company.locations.create!(city: "Denver")
```

Comment:
Would have loved to see you set this portion of the test up with a location belonging to many different cities versus the same city having multiple locations of the same name. This way, we only need to have a location with the name of Denver once in our database and each company that has that location can just reference its foreign key.